src/SmartComponents/CompletedTaskDetails/CompletedTaskDetails.js

```diff
@@ -1,7 +1,6 @@
 import React, { useState, useEffect } from 'react';
-import { useHistory, useParams } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
```

Comment:
this pulls out the id, but then the id isn't used. Maybe we can drop it?

Reply:
I believe the `id` is still being used here and in the `useEffect`

![image](https://github.com/RedHatInsights/tasks-frontend/assets/20592948/31d01bd9-9f3a-4dd9-91f8-63c81f1a16fb)